package.json

```diff
@@ -30,7 +31,9 @@
   "devDependencies": {
     "audio-generator": "^1.0.5",
     "audio-speaker": "^1.1.2",
-    "watchify": "^3.6.0"
+    "mic": "^2.1.1",
+    "watchify": "^3.6.0",
```

Comment:
we could use *budo*, but it's a lot more complex than watchify. And the example works by opening the `test.html` directly from a folder. No need to have a server to run it.